packages/block-library/src/code/utils.js

```diff
@@ -11,49 +16,12 @@ import { flow } from 'lodash';
  */
 export function escape( content ) {
 	return flow(
-		escapeAmpersands,
+		escapeEditableHTML,
 		escapeOpeningSquareBrackets,
 		escapeProtocolInIsolatedUrls
 	)( content || '' );
 }
 
-/**
- * Unescapes escaped ampersands, shortcodes, and links.
- *
- * @param {string} content Content with (maybe) escaped ampersands, shortcodes, and links.
- * @return {string} The given content with escaped characters unescaped.
- */
-export function unescape( content ) {
```

Comment:
👋 @ellatrix , can you reinstate `unescape` since it's used in https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/code/edit.native.js#L33 and is breaking the mobile tests at the moment? We can discuss what's the best way to remove the dependence to `unescape` too.

Reply:
Let’s adjust the escaping instead, no? Sorry for not seeing it

Reply:
I'd prefer if we lifted the breakage so other PRs currently blocked can proceed while we properly adjust to this.

Reply:
That will lift the breakage?

Reply:
Sorry, I'm having lunch but the PR comes in a bit.

Reply:
Thanks for the quick fix PR Ella! 🙇‍♂️